BridgeSDK/BridgeAPI/Categories/NSBundle+SBBAdditions.m

```diff
@@ -80,6 +80,10 @@ - (NSURL * _Nonnull)appStoreLinkURL
     return [NSURL URLWithString:[NSString stringWithFormat:@"http://appstore.com/%@", appName]];
 }
 
+- (NSURL * _Nonnull)privacyPolicyURL {
+    return [NSURL URLWithString:@"http://parkinsonmpower.org/mpower/privacy.html"];
```

Comment:
Don't think we should have specific mPower url links in BridgeSDK which is meant to be used by all apps. Can this be migrated up to the mPower bundle?